application-admintools-default/src/main/java/com/xwiki/admintools/internal/configuration/DefaultAdminToolsConfiguration.java

```diff
@@ -60,4 +62,10 @@ public List<String> getExcludedLines()
         return new ArrayList<>(
             List.of(this.mainConfiguration.getProperty(EXCLUDED_LINES, "NO_EXCLUDED_LINE").split(",")));
     }
+
+    @Override
+    public String getSpamSize()
```

Comment:
It's a bit odd to return a string for "size". Even if you need it as a string, its more intuitive to return an integer.